ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.5.yaml

```diff
@@ -65,6 +65,8 @@ tests:
   cron: 0 */2 * * *
   steps:
     cluster_profile: packet-assisted
+    env:
+      LEAK_THRESHOLD_SECONDS: 5:0:0
```

Comment:
I'm actually not sure why we have this job for 2.5 branch. I think it's only needed once per cluster-profile

Reply:
True, thought about it. But it's just gonna get cloned again next time a bot / person creates the next release config file

Reply:
Deleted them

Reply:
Right. Last time we pretty much copied everything, lots of it was redundant when considering the purpose of this branch